src/components/card/card.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import styled from 'styled-components'
 
 const Wrapper = styled.div(({ theme }) => `
-  width: 100%;
+  width: 40vw;
```

Comment:
I dont think we should change the card definition. This was to accommodate columns? Then better would be to put them in a grid or flex container when using cards, and let them fill 100% of their container. Otherwise cards are 40vw wide by default, which seems strange.

Reply:
Ahh right. I'll take it!